src/console/content.ts

```diff
@@ -406,56 +409,60 @@ class ConsoleContent extends Widget {
 
     // Create the new prompt.
     prompt = this._renderer.createPrompt(this._rendermime, this);
-    prompt.mimetype = this._mimetype;
+    prompt.model.mimeType = this._mimetype;
```

Comment:
Not a big deal, but we should probably keep the camelCasing of `this._mimetype` and `model.mimeType` consistent.